### PR TITLE
A series of patches we use in production

### PR DESCRIPTION
**Pull request by posix4e**
We needed this for testing purposes

### CODE REVIEW

File: sqs_listener/__init__.py [hunk 1]
```diff
@@ -48,18 +48,20 @@ def __init__(self, queue, **kwargs):
...
 
 
     def _initialize_client(self):
-        #sqs = boto3.client('sqs', region_name=self._region_name)
-
         # new session for each instantiation
         self._session = boto3.session.Session()
-        sqs = self._session.client('sqs', region_name=self._region_name)
+        ssl = True
+        if self._region_name == 'elasticmq':
```
Comment (jegesh) [inline sqs_listener/__init__.py]: What is the significance of `elasticmq` as a region name?  Are there other reserved names the script should be aware of?

Comment (e271828-) [reply sqs_listener/__init__.py]: @jegesh This is a common pattern when using https://github.com/adamw/elasticmq for testing. `elasticmq` is its default region and will never be a real AWS region, so this should be safe.

**Comment by jegesh** [comment]
@posix4e @e271828-  I still don't fully understand why this should be added to the main repo.  Is this a very common pattern that the library should support, or is it a specialized use case that should remain in the forked repo?

P.S. I apologize for the long delay in getting back to this

**Comment by e271828-** [comment]
@jegesh it's common enough to be worth merging, since it's a common testing pattern for serverless apps.

**Comment by jegesh** [comment]
@posix4e Merged and bumped pip version.  Thanks for the contribution

**Pull request merged**